src/System.Data.SqlClient/ref/project.json

```diff
@@ -5,7 +5,8 @@
     "System.IO": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Threading.Tasks": "4.0.0",
-    "System.Xml.ReaderWriter": "4.0.0"
+    "System.Xml.ReaderWriter": "4.0.0",
+	"System.Collections.NonGeneric": "4.0.0",
```

Comment:
Also nit: the whitespace is incorrect here (it is currently a tab but should be 4 spaces).